### ISSUE TITLE
Suggestion: Column-Filters

### ISSUE THREAD

**Issue by clangdev** [justinleewells/pogo-optimizer#188]
What about filtering columns?
Setting up sort order is nice but if one could additionally filter all pokemon with less than 80IV% would be pretty nice addon, what do you think?

**Comment by justinleewells** [comment]
Yeah, this could be cool. I'll think about how to work it into the app.
